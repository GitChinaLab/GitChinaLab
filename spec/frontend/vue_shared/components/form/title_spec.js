import { shallowMount } from '@vue/test-utils';
import TitleField from '~/vue_shared/components/form/title.vue';

describe('Title edit field', () => {
  let wrapper;

  function createComponent() {
    wrapper = shallowMount(TitleField);
  }

  beforeEach(() => {
    createComponent();
  });

  afterEach(() => {
    wrapper.destroy();
  });

  it('matches the snapshot', () => {
    expect(wrapper.element).toMatchSnapshot();
  });
});
