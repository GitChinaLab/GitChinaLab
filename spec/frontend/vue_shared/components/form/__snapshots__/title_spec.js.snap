// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title edit field matches the snapshot 1`] = `
<gl-form-group-stub
  label="Title"
  label-for="title-field-edit"
  labeldescription=""
  optionaltext="(optional)"
>
  <gl-form-input-stub />
</gl-form-group-stub>
`;
